### ISSUE TITLE
Best fork and pull requests

### ISSUE THREAD

**Issue by tablatronix** [dbushell/Nestable#163]
Since this is no longer active , can anyone recommend the best fork, or if there is a collection of preferred commits or pr's ?

The network is fairly large since last commit, almost to much to track.
At least a better way to track all meaningful commit changes would be useful.

**Comment by bigfoot90** [comment]
I'm using this one https://github.com/dbushell/Nestable/pull/143, it adds some events:
**beforeDragStart**, **dragStart**, **dragMove**, **dragEnd**, **beforeDragEnd**, **dragEnd**

**Comment by pwnash** [comment]
Has anyone offered to take ownership of the project?

**Comment by tablatronix** [comment]
ramonsmit, said he was creating a active fork, but 
1. its dead
2. its not a fork, so its pointless, and hard to track what was merged.

See my issues on that repo..

**Comment by tablatronix** [comment]
This seems useful, #156
on drop event with rejection capability, i wonder if it is compatible with other callback PRs.

Does #143 already allow for cancellation of drops?

**Comment by tablatronix** [comment]
If you need nested sets
https://github.com/tchapi/Nestable/commit/daabd980ebd9b6207ad8589593bcdcba14087668
https://github.com/RamonSmit/Nestable/commit/9fedbb55f31eff1ec0492f495f60669b097bb115

**Comment by tablatronix** [comment]
https://github.com/lukasoppermann/Nestable-fork
adds drag cloning, drag out delete, and jqueryui free scrolling. untested.

**Comment by bigfoot90** [comment]
@tablatronix yes, with ```feedback = {abort: true}``` you can deny an item to be dragged/dropped.
In the future it can be enhanced using ```event.stopPropagation()```, but currently I missing time to work on this.
